src/app/init.coffee

```diff
@@ -321,49 +325,56 @@ module.exports = class Init
         # Help :
         # initial_state: event: end_state
 
-        # Start application
         'init':
-            'startApplication': 'aDeviceLocale'
-            'startService': 'sDeviceLocale'
+            'startApplication': 'aConfigLoad'
+            'startService': 'sConfigLoad'
+
+        ########################################
+        # Start application
+        'aConfigLoad': 'loaded': 'aDeviceLocale'
         'aDeviceLocale': 'deviceLocaleSetted': 'aInitFileSystem'
-        'aInitFileSystem': 'fileSystemReady': 'aInitDatabase'
-        'aInitDatabase': 'databaseReady': 'aInitConfig'
-        'aInitConfig':
-            'configured': 'nPostConfigInit' # Normal start
-            'newVersion': 'migrationInit' # Migration
-            'notConfigured': 'fQuitSplashScreen' # First start
-            # First start error
-            'goTofDeviceName': 'f1QuitSplashScreen'
-            'goTofInitContacts': 'f2QuitSplashScreen'
-            'goTofInitCalendars': 'f3QuitSplashScreen'
-            'goTofUpdateIndex': 'f4QuitSplashScreen'
+        'aInitFileSystem': 'fileSystemReady': 'aQuitSplashScreen'
+        'aQuitSplashScreen': 'viewInitialized': 'aApplicationState'
+        'aApplicationState':
+            'default': 'fWizardWelcome'
+            'deviceCreated': 'fWizardFiles'
+            'appConfigured': 'fFirstSyncView'
+            'syncCompleted': 'nLoadFilePage'
```

Comment:
Add a state to restart application after first replication page when is terminated